docs/drafts/how-to-pin-version.md

```diff
@@ -0,0 +1,23 @@
+## How-to: Version Pin and Disable Automatic Updates
+
+To disable automatic updates, and pin the version of an extension, set `version` in the Catalog source to a specific version (e.g. 1.2.3).
+
+Example:
+
+```yaml
+apiVersion: olm.operatorframework.io/v1alpha1
+kind: ClusterExtension
+metadata:
+  name: argocd
+spec:
+  installNamespace: argocd
```

Comment:
There was a recent [PR](https://github.com/operator-framework/operator-controller/pull/1200) that put the `installNamespace` and `serviceAccount` fields under an `install` root field.

Reply:
Thank you! I've fixed it up ^^